pom.xml

```diff
@@ -295,7 +299,6 @@
                             <value>33849</value>
                         </property>
                     </systemProperties>
-                    <argLine>-Xmx1024m -XX:MaxPermSize=380m</argLine>
```

Comment:
`MaxPermSize` no longer has any effect on Java 11. The use of `argLine` in plugin POMs in general interferes with the plugin POM's automatic setting of `--add-opens` directives in `argLine` for Java 17 builds, and the setting of `-Xmx` didn't seem worthwhile enough to justify potential complications when moving to Java 17.